library/include/ck/library/utility/check_err.hpp

```diff
@@ -23,6 +23,74 @@
 namespace ck {
 namespace utils {
 
+template <typename ComputeDataType>
+double get_relative_threshold()
+{
+    using F8   = ck::f8_t;
+    using F16  = ck::half_t;
+    using BF16 = ck::bhalf_t;
+    using F32  = float;
+
+    int mantissa = 0;
+    if constexpr(is_same_v<ComputeDataType, F8>)
+    {
+        mantissa = NumericUtils<F8>::mant;
+    }
+    else if constexpr(is_same_v<ComputeDataType, F16>)
+    {
+        mantissa = NumericUtils<F16>::mant;
+    }
+    else if constexpr(is_same_v<ComputeDataType, BF16>)
+    {
+        mantissa = NumericUtils<BF16>::mant;
+    }
+    else if constexpr(is_same_v<ComputeDataType, F32>)
+    {
+        mantissa = NumericUtils<F32>::mant;
+    }
+    else
+    {
+        std::cout << "Warning: Unhandled ComputeDataType for setting up the relative threshold!"
+                  << std::endl;
+    }
+
+    return std::pow(2, -mantissa) * 0.5;
+}
+
+template <typename ComputeDataType>
+double get_absolute_threshold()
+{
+    using F8   = ck::f8_t;
+    using F16  = ck::half_t;
+    using BF16 = ck::bhalf_t;
+    using F32  = float;
+
+    auto expo = std::log2(std::abs(0.5));
```

Comment:
```suggestion
    auto expo = NumericUtils<ComputeDataType>::exp;
```

Reply:
This should be exp from maximum possible value. Absolute error depends on real values in tensors. Please add parameter to get_absolute_threshold to get it.